tests/python/test_cli.py

```diff
@@ -0,0 +1,85 @@
+import os
+import tempfile
+import unittest
+import xgboost
+import subprocess
+import numpy
+
+
+class TestCLI(unittest.TestCase):
+    template = '''
+booster = gbtree
+objective = reg:squarederror
+eta = 1.0
+gamma = 1.0
+seed = 0
+min_child_weight = 0
+max_depth = 3
+task = {task}
+model_in = {model_in}
+model_out = {model_out}
+test_path = {test_path}
+name_pred = {name_pred}
+
+num_round = 10
+data = {data_path}
+eval[test] = {data_path}
+'''
+
+    def test_cli_model(self):
+        curdir = os.path.normpath(os.path.abspath(os.path.dirname(__file__)))
+        project_root = os.path.normpath(
+            os.path.join(curdir, os.path.pardir, os.path.pardir))
+        data_path = "{root}/demo/data/agaricus.txt.train?format=libsvm".format(
+            root=project_root)
+        exe = os.path.join(project_root, 'xgboost')
```

Comment:
```suggestion
        exe = os.path.join(project_root, 'xgboost.exe' if sys.platform == 'win32' else 'xgboost')
```
(Import `sys` package at the top)

Executables in Windows have `.exe` extension. This should fix https://xgboost-ci.net/blue/organizations/jenkins/xgboost-win64/detail/PR-5535/1/pipeline